Sources/Action/Action.swift

```diff
@@ -30,7 +34,11 @@ public final class Action<Input, Element> {
 
     /// Errors aggrevated from invocations of execute().
     /// Delivered on whatever scheduler they were sent from.
-    public let errors: Observable<ActionError>
+    public let errors: Observable<Error>
+    
+    /// Errors when Action cannot execute due to disabled.
+    /// Delivered on whatever scheduler they were sent from.
```

Comment:
```suggestion
    /// Delivered on the scheduler they originated from.
```

Reply:
I just want to keep original code/comment that I don't touch